### PR TITLE
Pin orca docs

### PR DESCRIPTION
**Pull request by nicolaskruchten**
docs version of #2293

### CODE REVIEW

File: doc/python/ternary-plots.md [hunk 1]
```diff
@@ -33,7 +33,35 @@ jupyter:
     thumbnail: thumbnail/v4-migration.png
 ---
 
-### Basic Ternary Plot with Markers
+## Ternary Plots
```
Comment (emmanuelle) [inline doc/python/ternary-plots.md]: this part does not seem related to the rest of the PR but it seems ok...

**Comment by emmanuelle** [comment]
:dancer:

File: doc/python/ternary-plots.md (hunk 1 above)
Comment (nicolaskruchten) [reply doc/python/ternary-plots.md]: sorry, should have been targeted to `doc-prod`
